docs/pages/operations-guide.md

```diff
@@ -151,47 +177,53 @@ Plans have a fixed three-level hierarchy. Plans contain Phases, and Phases conta
+Another common/default Plan is the Recovery Plan, which handles bringing back failed tasks. The Recovery Plan listens for offers that can be used to bring back those tasks and then relaunches tasks against those offers.
 
-### Recovery Plan
+The Scheduler learns whether a task has failed by receiving Task Status updates from Mesos. Task Status updates can be sent during startup to let the scheduler know when a task has started running, to know when the task has exited successfully, or to know when the cluster has lost contact with the machine hosting that task.
 
-In addition to sending Offers as described above, Mesos will periodically send Task Status updates which notify the Scheduler about the state of its tasks. Task Status updates can be sent during startup to let the scheduler know when a task has started running, or know when the task has exited successfully, or when the cluster has lost contact with the machine hosting that task. As such the Scheduler needs to decide whether a given status update indicates a task which needs to be relaunched. When this is the case, the Scheduler will simply wait on the Offer cycle. We've mentioned the [Deployment Plan](#plans) earlier, but the Scheduler has many Plans for various operations, some included by default and others defined for custom service-specific behavior. Another common/default Plan is the Recovery Plan, which handles bringing back tasks which have failed. The Recovery Plan listens for offers which may be used to bring back those tasks, and then relaunches tasks against those offers.
+When it receives a Task Status update, the Scheduler decides whether a given update indicates a task that needs to be relaunched. When a task must be relaunched, the Scheduler will wait on the Offer cycle.
 
-In practice there are two types of recovery, permanent and temporary. The difference is mainly an issue of whether the task being recovered should stay on the same machine, and the side effects that result from that:
+#### Permanent and temporary recovery
+
+There are two types of recovery, permanent and temporary. The difference is mainly whether the task being recovered should stay on the same machine, and the side effects that result from that.
 
 - __Temporary__ recovery:
-    - Hiccup in the task or the host machine.
+    - Temporary recovery is triggered when there is a hiccup in the task or the host machine.
     - Recovery involves relaunching the task on the same machine as before.
     - Recovery occurs automatically.
     - Any data in the task's persistent volumes survives the outage.
     - May be manually triggered by a `pods restart` command.
 - __Permanent__ recovery:
-    - Permanent failure of the host machine, no reason to stick around.
+    - Permanent recovery is triggered when the host machine fails permanently.
```

Comment:
Maybe something like "Permanent recovery may be requested when the host machine fails permanently or when the host machine is scheduled for downtime": be clear that it's not automatic, and point out that it may also be performed for scheduled outages